tests/proxy_test.go

```diff
@@ -233,7 +233,7 @@ func testVm(namePrefix string) *kubevirtcorev1.VirtualMachine {
 			Namespace:    testNamespace,
 		},
 		Spec: kubevirtcorev1.VirtualMachineSpec{
-			Running: pointer.BoolPtr(true),
+			Running: pointer.Bool(true),
```

Comment:
Isn't this package deprecated and replaced by `ptr.To`?

Reply:
done